pilot/pkg/networking/core/v1alpha3/cluster.go

```diff
@@ -963,6 +963,19 @@ func applyUpstreamTLSSettings(opts *buildClusterOpts, tls *networking.TLSSetting
 					},
 				},
 			}
+		} else if tls.Mode == networking.TLSSettings_MUTUAL {
+			// These are certs being mounted from within the pod. Rather than reading directly in Envoy,
```

Comment:
what about the ca certs?

Reply:
I need to add this, good catch.

/hold

should probably also decide if we can do https://github.com/envoyproxy/envoy/issues/10387 before investing time in this